codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSSmokeTestGenerator.kt

```diff
@@ -26,11 +26,15 @@ class AWSSmokeTestGenerator(
     )
 
     override fun getServiceName(): String {
-        return "AWS" + ctx.service.getTrait(ServiceTrait::class.java).get().sdkId.toUpperCamelCase()
+        val serviceTrait = ctx.service.getTrait(ServiceTrait::class.java).orElse(null)
+        val sdkId = serviceTrait?.sdkId?.toUpperCamelCase() ?: "DefaultService"
+        return "AWS$sdkId"
     }
 
     override fun getClientName(): String {
-        return ctx.service.getTrait(ServiceTrait::class.java).get().sdkId.toUpperCamelCase().removeSuffix("Service") + "Client"
+        val serviceTrait = ctx.service.getTrait(ServiceTrait::class.java).orElse(null)
+        val sdkId = serviceTrait?.sdkId?.toUpperCamelCase()?.removeSuffix("Service") ?: "Default"
+        return "${sdkId}Client"
```

Comment:
I think it's far from ideal to have these changes just to accommodate codegen'ing for protocol test model that doesn't have sdkId. Did you create the follow-up ticket for re-locating RPCv2 CBOR protocol as discussed last week btw?

Reply:
relocating RPCv2 CBOR to smithy-swift wouldn't change the need for this. Its because the protocol tests themselves do not have a service trait defined. There is a ticket for relocating RPC to smithy-swift but that change should be purely cosmetic as I mentioned.